.circleci/config.yml

```diff
@@ -18,6 +18,11 @@ jobs:
 
     steps:
     - checkout
+    - run:
+        name: create and set the default keychain
+        command: |
+          security create-keychain -p "" temporary
+          security default-keychain -s temporary
```

Comment:
@daviwil I bet this would fix our problems on VSTS too 🤔

Reply:
Sweet! I'll give it a shot today